instance/tests/models/test_log_entry.py

```diff
@@ -168,19 +168,19 @@ def test_log_delete(self):
         server1.logger.info('Line #2, on server 1')
         server2.logger.info('Line #3, on server 2')
 
-        self.assertEqual(LogEntry.objects.count(), 3)
+        self.assertEqual(LogEntry.objects.count(), 4)
         # Delete server 1:
         server1_id = server1.pk
         server1.delete()
         # Now its log entry should be deleted:
         entries = LogEntry.objects.order_by('pk').all().values_list('text', flat=True)
         for entry_text in entries:
             self.assertNotIn('Line #2', entry_text)
-        self.assertIn('Line #1, on instance', entries[0])
-        self.assertIn('Line #3, on server 2', entries[1])
+        self.assertIn('Line #1, on instance', entries[1])
```

Comment:
The new line comes from the implicit creation of the default load balancer when calling `OpenEdXInstanceFactory`. This test function verifies that log lines for a server get deleted if the server gets deleted, so I didn't feel the line resulting from something happening in the `setUp()` method is related.  I'll add an assertion just to prevent people from wondering what that line might be.